code/datums/components/plumbing/_plumbing.dm

```diff
@@ -213,3 +213,11 @@
 /datum/component/plumbing/tank
 	demand_connects = WEST
 	supply_connects = EAST
+
+/datum/component/plumbing/hydroponics
```

Comment:
Can this be modularized please?

Reply:
Moved it to modular_splurt. Is that all you mean by that?